### ISSUE TITLE
New Rule: Enforce property shorthand

### ISSUE THREAD

**Issue by DanPurdy** [sasstools/sass-lint#259]
For more info see #15 

This rule should enforce the use of shorthand properties where possible such as the following

```scss
.selector {
  border-width: 1px;
  border-style: solid;
  border-color: #fff;
}

```

Should actually be

```scss
.selector {
  border: 1px solid #fff;
}
```
